docs/resources/servicestage_component.md

```diff
@@ -0,0 +1,135 @@
+---
+subcategory: "ServiceStage"
+---
+
+# huaweicloud_servicestage_component
+
+This resource is used to manage a component under specified application within HuaweiCloud ServiceStage service.
+
+## Example Usage
+
+### Create a Web component using GitHub repository
+
+```hcl
+variable "application_id"
+variable "component_name"
+variable "token_auth_name"
+variable "repo_url"
+variable "domain_name"
+variable "cluster_id"
+
+resource "huaweicloud_servicestage_component" "test" {
+  application_id = var.application_id
+  name           = var.component_name
+  type           = "Webapp"
+  runtime        = "Nodejs14"
```

Comment:
pls add this link , https://support.huaweicloud.com/intl/en-us/usermanual-servicestage/servicestage_user_0411.html

Reply:
done.